src/Ng/CommandHelpers.cs

```diff
@@ -178,7 +178,8 @@ public static StorageFactory CreateSuffixedStorageFactory(string suffix, IDictio
                 { Arguments.StorageAccountName, Arguments.StorageAccountName + suffix },
                 { Arguments.StorageKeyValue, Arguments.StorageKeyValue + suffix },
                 { Arguments.StorageContainer, Arguments.StorageContainer + suffix },
-                { Arguments.StoragePath, Arguments.StoragePath + suffix }
+                { Arguments.StoragePath, Arguments.StoragePath + suffix },
+                { Arguments.StorageSuffix, Arguments.StorageSuffix + suffix }
```

Comment:
Yes, I think that would be better to change it. I can do that in a future change.